app/models/dormitory.rb

```diff
@@ -0,0 +1,9 @@
+class Dormitory < ApplicationRecord
+    attr_accessor :id, :name, :address
+
+    def initialize(id, name, address)
```

Comment:
I don't think we need these accessors and initializer here, we should have it all handled by rails